sdk/security_keyvault/src/keys/models.rs

```diff
@@ -406,6 +406,10 @@ pub struct EncryptResult {
 
 #[derive(Debug, Deserialize)]
 pub struct GetRandomBytesResult {
-    #[serde(rename = "value")]
-    pub result: String,
+    #[serde(
+        rename = "value",
+        serialize_with = "ser_base64",
```

Comment:
It's actually base64url: https://github.com/Azure/azure-rest-api-specs/blob/be4adb7c2681e82c81c25902bb5e4964229a79a2/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/keys.json#L1230

Reply:
Got it. Thx for the reminder.